php/class-coauthors-guest-authors.php

```diff
@@ -149,6 +149,11 @@ function filter_post_updated_messages( $messages ) {
 		global $post;
 
 		$guest_author = $this->get_guest_author_by( 'ID', $post->ID );
+
+		if( !$guest_author ) {
```

Comment:
We should follow WordPress PHP coding standards for spacing: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage